Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBiasHI_cff.py

```diff
@@ -26,7 +26,8 @@
 
 import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi
 
-seqALCARECOHcalCalMinBias = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB*hbherecoNoise*hfrecoNoise*hfrecoMBNZS*horecoNoise)
+seqALCARECOHcalCalMinBiasDigi = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB)
```

Comment:
@bsunanda what is the purpose of dividing the whole sequence into two parts? This question applies to other file as well?